unpick-format-utils/src/main/java/daomephsta/unpick/constantmappers/datadriven/tree/UnpickV3Visitor.java

```diff
@@ -0,0 +1,12 @@
+package daomephsta.unpick.constantmappers.datadriven.tree;
+
+public abstract class UnpickV3Visitor {
```

Comment:
Is there any reason for this to be a class instead of an itf?

Reply:
Ah, the whole class vs interface dilemma! It's subjective but there are no abstract methods in this class so it doesn't feel very interface-y to me.